src/main/scala/jigg/util/JSONUtil.scala

```diff
@@ -116,6 +120,64 @@ object JSONUtil {
   }
 
   def writeToJSON[T <: Node](x: T, os: BufferedWriter): Unit = {
-    os.write(toJSON(x).toString)
+    os.write(toJSON(x))
```

Comment:
This separation seems to be not necessary.
I'll remove them from `JSONUtil` , and implement the output sequences into `Pipeline.run` and `Pipeline.runFromStdin`